src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelMetadata.cs

```diff
@@ -363,7 +363,8 @@ internal IReadOnlyList<ModelMetadata> BoundProperties
     public abstract int Order { get; }
 
     /// <summary>
-    /// Gets the text to display as a placeholder value for an editor.
+    /// Gets the text to display as a placeholder value for an editor,
+    /// corresponding to <see cref="System.ComponentModel.DataAnnotations.DisplayAttribute.Prompt" />.
```

Comment:
This should go in `DefaultModelMetadata`. ModelMetadata is an abstraction and isn't tied to a specific data source.

Reply:
So is [DisplayFor](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.rendering.ihtmlhelper-1.displayfor), which did not stop us from writing
> Custom templates are found under a `DisplayTemplates` folder.